.upptimerc.yml

```diff
@@ -25,6 +25,11 @@ sites:
     icon: https://www.steuerlotse-rente.de/favicon.ico
     maxResponseTime: 5000
 
+  - name: eid
+    url: https://eid.digitalservicebund.de/actuator/health
+    icon: https://eid.digitalservicebund.de/images/favicon.png
+    maxResponseTime: 5000
+
```

Comment:
do we need an assignee? 
https://github.com/digitalservicebund/uptime-monitor/blob/3118b1017f49a5505e274b3b089dcd2971faeea1/.upptimerc.yml#L20-L21

Reply:
Not yet. Once things are wired up in the background